packages/koenig-lexical/src/components/ui/CardWrapper.jsx

```diff
@@ -13,6 +22,26 @@ export const CardWrapper = React.forwardRef(({
     children,
     ...props
 }, ref) => {
+    const wrapperClass = () => {
+        if ((wrapperStyle === 'wide') && (isEditing || isSelected)) {
+            return '!-mx-3 !px-3';
+        } else if (((wrapperStyle === 'code-card') && isEditing)) {
+            return '-mx-6';
+        } else if (wrapperStyle === 'wide') {
+            return 'hover:-mx-3 hover:px-3';
+        } else {
+            return 'border';
+        }
+    };
+
+    const className = [
+        'relative border-transparent caret-grey-800',
+        isSelected && !isDragging ? 'shadow-[0_0_0_2px] shadow-green' : '',
+        !isSelected && !isDragging ? 'hover:shadow-[0_0_0_1px] hover:shadow-green' : '',
+        CARD_WIDTH_CLASSES[cardWidth] || '',
+        wrapperClass()
+    ].join(' ');
```

Comment:
Just split these out to help my own understanding. Could merge them again, not sure if there's a particular reason to have them all in one string

Reply:
Don't think so – much more readable this way, thanks!